packages/keystone/cipher.js

```diff
@@ -0,0 +1,157 @@
+const crypto = require('crypto')
+
+const { isEmpty } = require('lodash')
+
+// since data is converted in hex, ':' shouldn't be in it
+const SEPARATOR = ':'
+const SUPPORTED_MODES = ['cbc', 'ctr', 'cfb', 'ofb', 'ecb']
+const SUGGESTIONS = {
+    'ecb': 'WARNING! Don\'t use "ecb" in real application as it is insecure!',
+    'cfb': 'Please, consider using "ctr" or "cbc"',
+    'ofb': 'Please, consider using "ctr" or "cbc"',
+}
+
+class CipherManager {
+
+    /** @type {{[version: string]: {cipher: string, secret: string, ivLength: number}}} */
+    versions = {}
+    currentVersionKey
+    
+    /**
+     * @param {Array<{version: string, cipher: string, secret: string}>} options
+     * @param {string?} currentVersionKey
+     */
+    constructor (options, currentVersionKey) {
```

Comment:
it's better to use more context friendy naming for argumens

Reply:
You can ask GPT for advice <img width="843" alt="image" src="https://github.com/user-attachments/assets/29dc9ed1-c388-4741-99af-9cae0dcdd450">

Reply:
Maybe names became a little bit better now